README.md

```diff
@@ -320,14 +319,14 @@ blank line.
 
 ## Hash arguments
 
-You can use Hash value not unlike Arrays. It's also multivalue type but
-you have to specify a key:value pairs like this.
+You can't use a Hash value like an Array. It's also a multivalue type but
```

Comment:
Actually it should have been: `You can use a Hash value similar to how you use an Array.`